dcos/dcos-ui
# [spike] DCOS_OSS-401: Incorrect error message for priority when adding a new repo

Covered the case of having 0 repositories.

Closes https://jira.mesosphere.com/browse/DCOS_OSS-401

## Testing
1. In `plugins/catalog/src/js/repositories/components/AddRepositoryFormModal.js` change line 29 to be `const noRepositories = numberOfRepositories !== 0;` or altrenatively delete all package repositories on your cluster (just remember to save them so that you can re-add them).
2. Go to Settings > Package Repositories and click the button in the top right corner to add a new repository.
3. Verify that the field is disabled and shows 0.

## Trade-offs
@lsamimi , what do you think about the current solution?

Not sure if we should add "0" to the message catalog.

Edit: Also added labels.

## Dependencies
None.

## Screenshots

### Before (from jira)
![screen shot 2016-09-22 at 1 11 11 pm](https://user-images.githubusercontent.com/40791275/49645226-bb6bbe80-fa23-11e8-90ea-6f86d661f55c.png)

### After
![screenshot from 2018-12-10 09-54-33](https://user-images.githubusercontent.com/40791275/49718524-07587680-fc63-11e8-84ea-29bd2e01665b.png)
